.npmignore

```diff
@@ -1 +0,0 @@
-bench
```

Comment:
Then you have to remember to add new test files to the `files` field. We're not putting secrets or password files in this repo, so there's no real risk of accidentally publishing something sensitive so the blacklist (npmignore) approach is actually simpler IMO